site/public/js/stat-page.js

```diff
@@ -0,0 +1,67 @@
+$('td').click(function () {
```

Comment:
You will need to add `/* global sortTable, escapeSpecialChars, buildCourseUrl */` to get this file to pass the linter as those  functions are defined elsewhere.

Reply:
just to understand you clearly, do you mean that I should implement them from scratch or import them from a file somewhere